design.html

```diff
@@ -0,0 +1,1145 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <title>Newsweek-replica</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+        integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="main.css" />
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/57e597c65c.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Podkova:wght@800&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <section class="top-navbar navbar navbar-expand sticky-top  m-0 p-0 ">
+      <h3>ty</h3>
+<div class="container-fluid  font-weight-bolder middle d-flex justify-content-between align-items-center ">
+    <a class="nav-item nav-link text-white ml-0 d-lg-inline d-none" href="" ><img class="climate" src="./assets/things/climate.jpg"    alt="climate" ><span>  26° Oaxaca, MX ></span><br>
+    <span > Tue, July 02, 2020</span> </a>
+    <a class=" logo nav-item nav-link float-sm-left float-md-left text-white  m-0" href="">Newsweek</a>
+    <article class=" login-subs mr-0 align-items-center  d-flex nav-item nav-link text-white " ><a class="d-lg-flex d-md-flex text-white d-none" href="">LOGIN</a><i class="fas d-sm-none fa-user"></i> <button type="button" class=" subscribe-btn border d-sm-flex d-md-flex   btn  font-weight-bold " >SUBSCRIBE <i class="fas fa-angle-right"></i></button><h3>ty</h3></article>
+</div>
+<i class="fas d-lg-none fa-bars"></i>
+</section>
+<div  class="little-corner d-lg-flex d-none position  ">
+<span class="corner  "></span></div>
+    <nav class=" container-fluid white-navbar navbar navbar-expand-sm w-100  font-weight-bold bg-white d-lg-flex d-none position-relative" >
+    <a class=" nav-item nav-link text-body border-right" href="">U.S.</a>
+    <a class=" nav-item nav-link text-body border-right" href="">World </a>
+   <a class="nav-item nav-link text-body border-right" href="">Business </a>
+    <a class="nav-item nav-link text-body border-right" href="">Tech & Science </a>
+    <a class="nav-item nav-link text-body border-right"  href="">Culture </a>
+    <a class="nav-item nav-link text-body border-right" href="">Newsgeek </a>
```

Comment:
### Mobile Section

- [ ] The page does not have a margin, and it is collapsing in this screen size (The navbar and the Top Story section)

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88102559-27a62200-cb76-11ea-88e5-b4ddb8da0d1d.png) | ![image](https://user-images.githubusercontent.com/61479648/88102659-502e1c00-cb76-11ea-9d9d-9b10fe51b1a9.png) |

- [ ] The page breaks at 690px screen size. Kindly fix that as well.

![image](https://user-images.githubusercontent.com/61479648/88103024-d6e2f900-cb76-11ea-8e2b-88a3ca9a9334.png)